.github/ISSUE_TEMPLATE/4_documentation_change.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Documentation change request
```

Comment:
Please capitalize the header row of the table: about, title, etc.

Please add a period to the end of full sentences: Report documentation issue...

Reply:
Done.